src/IotCentral/IotCentral/ChangeLog.md

```diff
@@ -19,12 +19,13 @@
 -->
 ## Upcoming Release
 
-## Version 0.7.4
+## Version 0.7.5
+* Allow pricing information to be updated using Set-AzIoTCentralApp command.
```

Comment:
Please put the change log of the upcoming release under `## Upcoming Release`. Do not add new versions.

Check the top of this file for an exmaple

Reply:
@isra-fel  Oops,  fixed.